.github/workflows/BuildNDeployDev.yml

```diff
@@ -0,0 +1,80 @@
+# This workflow will checkout the contents of the `S3Dev` branch of this repo, remove all files in the current S3 bucket, and push the contents
+# to the bucket. Once the contents of the S3 bucket has been updated, a call will be made to invalidate the cache in the CloudFront distribution.
+# The CloudFront distribution is configured to pull the contents from the S3 bucket once an `invalidation` request is sent.
+
+name: Build and Deploy Static Site
+
+on: # run this workflow when a push has been made to `Jekyll-Alf` branch
+  push:
+    branches:
+      - Jekyll-Alf
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:   
+ ###########################################################################################################
+ #      This is the CI portion
+ ###########################################################################################################
+        - uses: actions/checkout@v2
+        - uses: actions/setup-ruby@v1
+          with:
+            ruby-version: '2.6' # Version range or exact version of a Ruby version to use, using semvers version range syntax.
+       
+
+        - uses: actions/cache@v2
+          with:
+            path: vendor/bundle
+            key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+            restore-keys: |
+              ${{ runner.os }}-gems-
```

Comment:
Add comment section to explain a bit more detail about this.

Reply:
@Paz-X  create an issue to come back and add appropriate comments as we discussed.